lib/index.js

```diff
@@ -11,6 +11,8 @@ var assertAccessibility = (tagName, props, children, log) => {
 
   var propTests;
   for (var propName in props) {
+    // There's no need to check props that are null or undefined.
```

Comment:
Seriously? You don't want want comments in your code? This whole file is already difficult to read. There's no reason to force people to go read the tests to figure out how/why something works.